packages/@okta/vuepress-site/docs/guides/index.md

```diff
@@ -14,7 +15,6 @@ guides:
  - common-hook-set-up-steps
  - configure-access-policy
  - configure-native-sso
- - configure-signon-policy
```

Comment:
You might not want to remove this, as it's used in your other PR 3057?

Reply:
since there is no longer content for configure-signon-policy, but rather for archive-configure-signon-policy - it errors if I leave this in the index file.